test/unit/rest/DataQueryEndpoints.test.js

```diff
@@ -171,6 +187,86 @@ describe('DataQueryEndpoints', () => {
         })
     })
 
+    describe('Getting last events (signed messages)', () => {
```

Comment:
Same notion here as with `WebsocketServer`: what is being tested? Do `DataQueryEndpoints` behave differently depending on whether the messages are signed or not? I would assume no, since there are not even any changes in the `DataQueryEndpoints.js` file. I don't think these new tests are adding any coverage in the `DataQueryEndpoints` file.

If we're testing classes that are not concerned about whether messages or signed or unsigned, it doesn't really matter what kind of messages we use to test them.